src/Admin/AbstractAdmin.php

```diff
@@ -1297,7 +1296,7 @@ public function setFilterPersister(?FilterPersisterInterface $filterPersister =
 
     public function getMaxPerPage(): int
     {
-        $sortValues = $this->getModelManager()->getDefaultSortValues($this->class);
+        $sortValues = $this->getDefaultSortValues();
```

Comment:
IMHO this kind of changes should be done in a separate commit and do not resolve merging conflicts and `NEXT_MAJOR` comments in the same commit (note that I'm not requesting changes, just a comment).

Reply:
I only change this because I had a conflict on this method.

There was multiple NEXT_MAJOR resolved in these methods, so every time it's touched in 3.x it creates conflict in master. It was easier to me to directly fix then.

Reply:
If it's ok for you, I need an approval to merge this. ;)